docs/coding_standards.md

```
@@ -0,0 +1,7 @@
+# Standard:
```

Markdownlint

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)